### PR TITLE
mysqlcluster: Support automatic rebuild of nodes by label.

### LINKED ISSUE #385: [feature] Support the more elegant way to rebuild node.
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

- Add a label to the node that needs to be rebuilt, then operator reconstruct the node according to the label.

- Automatically rebuild nodes for certain specific circumstances

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by acekingke**
### What type of PR is this?

/enhancement
### Which issue(s) this PR fixes?

Fixes #385 

### What this PR does?

Summary:
1. track lable `rebuild` and rebuild the pod

### CODE REVIEW

File: mysqlcluster/syncer/status.go [hunk 1]
```diff
@@ -95,6 +96,12 @@ func (s *StatusSyncer) Sync(ctx context.Context) (syncer.SyncResult, error) {
 	// get ready nodes.
 	var readyNodes []corev1.Pod
 	for _, pod := range list.Items {
+		if pod.ObjectMeta.Labels["rebuild"] == "true" {
```
Comment (runkecheng) [inline mysqlcluster/syncer/status.go:99]: Define the content of the label into a global variable

File: mysqlcluster/syncer/status.go [hunk 2]
```diff
@@ -183,6 +190,31 @@ func (s *StatusSyncer) updateClusterStatus() apiv1alpha1.ClusterCondition {
 	return clusterCondition
 }
 
+func (s *StatusSyncer) AutoRebuild(ctx context.Context, pod *corev1.Pod) error {
+	ordinal, err := utils.GetOrdinal(pod.Name)
+	if err != nil {
+		return err
+
+	}
+	if err := s.cli.Delete(ctx, pod); err != nil {
```
Comment (runkecheng) [inline mysqlcluster/syncer/status.go:199]: Set unhealthy and update before delete pod

**Review by runkecheng — CHANGES_REQUESTED** [review]

**Review by runkecheng — APPROVED** [review]

**Review by andyli029 — APPROVED** [review]
1. Add command
2. Fix the commit log

**Review by runkecheng — APPROVED** [review]

**Review by andyli029 — COMMENTED** [review]
LGTM.

**Pull request merged**